opennms-doc/guide-admin/src/asciidoc/text/operation/config-tester.adoc

```diff
@@ -0,0 +1,27 @@
+
+// Allow GitHub image rendering
+:imagesdir: ../../../images
+
+[[ga-operation-config-tester]]
+=== Configuration Tester
+
+To identify configuration problems there is a config-tester located in `$OPENNMS_HOME/bin/`.
+The `config-tester` will be run while OpenNMS starts and prints the issues into `output.log`.
```

Comment:
```suggestion
The `config-tester` runs while OpenNMS starts and checks the following: x, y, z. 
It prints issues into `output.log`.
```
@ronny-trommer , could you add what the config-tester tests?

Reply:
The list of supported configuration files that can be tested is shown with `-l, --list`. It is a pretty long list of files and might be tedious to keep them maintained here. Here is a list which is currently supported:

```
Supported configuration files:
    ackd-configuration.xml
    actiond-configuration.xml
    ami-config.xml
    availability-reports.xml
    bsf-northbounder-configuration.xml
    categories.xml
    chart-configuration.xml
    collectd-configuration.xml
    database-reports.xml
    database-schema.xml
    datacollection-config.xml
    destinationPaths.xml
    discovery-configuration.xml
    drools-northbounder-configuration.xml
    eventconf.xml
    eventd-configuration.xml
    exclude-ueis.properties
    email-northbounder-configuration.xml
    groups.xml
    http-datacollection-config.xml
    jasper-reports.xml
    javamail-configuration.properties
    javamail-configuration.xml
    jdbc-datacollection-config.xml
    jmx-config.xml
    jmx-datacollection-config.xml
    xml-datacollection-config.xml
    ksc-performance-reports.xml
    microblog-configuration.xml
    notifd-configuration.xml
    notificationCommands.xml
    notifications.xml
    opennms-datasources.xml
    opennms.properties
    poll-outages.xml
    poller-configuration.xml
    provisiond-configuration.xml
    reportd-configuration.xml
    response-adhoc-graph.properties
    response-graph.properties
    rrd-configuration.properties
    rtc-configuration.xml
    rws-configuration.xml
    scriptd-configuration.xml
    service-configuration.xml
    site-status-views.xml
    snmp-adhoc-graph.properties
    snmp-config.xml
    snmp-graph.properties
    snmptrap-northbounder-configuration.xml
    statsd-configuration.xml
    surveillance-views.xml
    syslogd-configuration.xml
    syslog-northbounder-configuration.xml
    telemetryd-configuration.xml
    threshd-configuration.xml
    thresholds.xml
    tl1d-configuration.xml
    translator-configuration.xml
    trapd-configuration.xml
    users.xml
    vacuumd-configuration.xml
    viewsdisplay.xml
    wmi-config.xml
    wmi-datacollection-config.xml
    vmware-cim-datacollection-config.xml
    vmware-datacollection-config.xml
    vmware-config.xml
    wsman-config.xml
    wsman-datacollection-config.xml
    opennms.properties.d
    imports
    foreign-sources
```